### PR TITLE
#26 added Amazon storage

### PR DESCRIPTION
**Pull request by asaen**

### CODE REVIEW

**Comment by karato** [comment]
I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by karato** [comment]
@carlosmiranda review it

File: src/main/java/com/nerodesk/AmazonStorage.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import javax.validation.constraints.NotNull;
+
+/**
+ * Amazon S3 storage.
+ *
+ * @author Alexey Saenko (alexey.saenko@gmail.com)
+ * @version $Id$
+ * @since 0.2
+ */
+public final class AmazonStorage implements Storage {
```
Comment (carlosmiranda) [inline src/main/java/com/nerodesk/AmazonStorage.java:51]: Let's mark this class with `@ToString` and `@EqualsAndHashCode`, for the latter targetting the fields `region` and `bucket`.

File: pom.xml [hunk 2]
```diff
@@ -133,6 +133,15 @@
             <artifactId>jcabi-manifests</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.jcabi</groupId>
+            <artifactId>jcabi-s3</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk</artifactId>
+            <version>1.7.13</version>
```
Comment (carlosmiranda) [inline pom.xml:142]: Is there any particular reason why we aren't using a later version here? I believe the AWS Java SDK is now up to 1.9.x.

Comment (asaen) [inline pom.xml:142]: We cannot use it since `jcabi-s3` is dependent on version 1.7.13

**Comment by asaen** [comment]
@carlosmiranda thank you, fixed, please take a look once more.

**Comment by carlosmiranda** [comment]
@rultor merge please

**Comment by ggajos** [comment]
@rultor merge please

**Comment by karato** [comment]
@rultor deploy

**Comment by karato** [comment]
@carlosmiranda Thanks a lot, I just topped your account for **15 mins**, transaction ID `52938830`. +15 to your rating, your total score is [+3507](http://www.netbout.com/b/35023?open=rating)

**Pull request merged**